cypress/integration/slot_booking_bbmp.js

```diff
@@ -2,45 +2,46 @@ describe("My First Test", () => {
   it("Open Cowin App!", () => {
     cy.viewport(1000, 1000);
     cy.visit("https://selfregistration.cowin.gov.in/");
-    cy.get("input#mat-input-0").type("<your_mobile_number>");
+    cy.get("input#mat-input-0").type("9968726089");
```

Comment:
You can remove your Phone Number :) ... I am going to have the phone number through env variables

Reply:
awesome :)